source/user-manual/reference/ossec-conf/global.rst

```diff
@@ -18,6 +18,7 @@ Options
 -------
 
 - `alerts_log`_
+- `compress_alerts`_
```

Comment:
This setting is not used. Please remove it from reference.

Reply:
Removed as requested.